pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -215,8 +215,21 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		newServerHost = homeWorkspace.Spec.URL
-
+		u, homeCluster, err := pluginhelpers.ParseClusterURL(homeWorkspace.Spec.URL)
+		if err != nil {
+			return fmt.Errorf("home URL %q does not point to a workspace", homeWorkspace.Spec.URL)
```

Comment:
`fmt.Errorf("home URL %q does not point to a workspace: %w", homeWorkspace.Spec.URL, err)`